### PR TITLE
Issue #1048

### PR DESCRIPTION
**Pull request by iwishiwasaneagle**
#1048

### CODE REVIEW

File: build_scripts/ubuntu_sim_common_deps.sh [hunk 1]
```diff
@@ -31,25 +31,8 @@ sudo -H pip install pandas jinja2 pyserial pyyaml
...
-    tar -xzf eprosima_fastrtps-1-7-1-linux.tar.gz eProsima_FastRTPS-1.7.1-Linux/
-    tar -xzf eprosima_fastrtps-1-7-1-linux.tar.gz requiredcomponents
-    tar -xzf requiredcomponents/eProsima_FastCDR-1.0.8-Linux.tar.gz
-    cpucores=$(( $(lscpu | grep Core.*per.*socket | awk -F: '{print $2}') * $(lscpu | grep Socket\(s\) | awk -F: '{print $2}') ))
-    (cd eProsima_FastCDR-1.0.8-Linux && ./configure --libdir=/usr/lib && make -j$cpucores && sudo make install)
-    (cd eProsima_FastRTPS-1.7.1-Linux && ./configure --libdir=/usr/lib && make -j$cpucores && sudo make install)
-    rm -rf requiredcomponents eprosima_fastrtps-1-7-1-linux.tar.gz
-    popd
-fi
+# Install FastRTPS 1.7.1 and FastCDR-1.0.8 by following the instructions below
```
Comment (TSC21) [inline build_scripts/ubuntu_sim_common_deps.sh]: No need. Please remove it as this is not part of a the sim common dependencies.

**Review by TSC21 — CHANGES_REQUESTED** [review]

**Review by TSC21 — APPROVED** [review]

**Comment by TSC21** [comment]
Thanks!

**Comment by TSC21** [comment]
@iwishiwasaneagle ah can you also remove line 8?

**Comment by iwishiwasaneagle** [comment]
Accidentally closed PR, reopened.

**Comment by TSC21** [comment]
Thanks perfect!

**Comment by hamishwillee** [comment]
FYI we only have this script to support easy setup of the ROS melodic environment. @TSC21 It might be worth updating ubuntu.sh such that we can use it as the base for setting up ROS - ie run this script, then follow the ROS instructions. Off the top of my head, that would mean having a an optional flag to remove some of the gazebo installation.

**Pull request merged**